src/ngraph/op/fused/layer_norm.hpp

```diff
@@ -0,0 +1,140 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/node.hpp"
+#include "ngraph/op/op.hpp"
+#include "ngraph/op/util/fused_op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        /// \brief Layer Normalization
+        ///
+        class LayerNorm : public ngraph::op::util::FusedOp
+        {
+        public:
+            NGRAPH_API
```

Comment:
These need to switch over to `type_description` form in .hpp and .cpp. See `abs.hpp` and `abs.cpp`

Reply:
I've made the changes.